php/meterpreter/ext_server_stdapi.php

```diff
@@ -1266,6 +1266,37 @@ function stdapi_registry_set_value($req, &$pkt) {
 }
 }
 
+if (!function_exists('stdapi_net_config_get_arp_table')) {
+if (is_linux()) {
+    register_command('stdapi_net_config_get_arp_table');
```

Comment:
The command ID needs to be included here

```suggestion
    register_command('stdapi_net_config_get_arp_table', COMMAND_ID_STDAPI_NET_CONFIG_GET_ARP_TABLE);
```